dev-packages/e2e-tests/test-applications/aws-serverless-esm/playwright.config.ts

```diff
@@ -0,0 +1,27 @@
+import { getPlaywrightConfig } from '@sentry-internal/test-utils';
+
+// Fix urls not resolving to localhost on Node v17+
+// See: https://github.com/axios/axios/issues/3821#issuecomment-1413727575
+import { setDefaultResultOrder } from 'dns';
+setDefaultResultOrder('ipv4first');
+
+const eventProxyPort = 3031;
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config = getPlaywrightConfig(
+  { startCommand: '' },
+  {
+    /* Run your local dev server before starting the tests */
+    webServer: [
+      {
+        command: `node start-event-proxy.mjs && pnpm wait-port ${eventProxyPort}`,
+        port: eventProxyPort,
+        stdout: 'pipe',
+      },
+    ],
+  },
+);
```

Comment:
Is this needed? The base config is already doing this ootb.

Reply:
Right now, it somewhat is necessary because the shared playwright config always expects an additional `startCommand` to start an application/development server. For our AWS Lambda tests, we don't need this because we invoke the function from within the test file. I'll open a separate PR to make `startCommand` optional since this concerns all e2e tests relying on the shared config.

Reply:
Ah, understood. Thanks for the explanation!

Reply:
PR to refactor: https://github.com/getsentry/sentry-javascript/pull/12842